msteams-platform/sbs-teams-conversation-bot.yml

```diff
@@ -70,16 +70,15 @@ items:
     1. Enter bot handle name in **Bot handle** field.
     1. Select your **Subscription** from the dropdown list.
     1. Select your **Resource group** from the dropdown list.
-
-        To create a new resource group, select **Create new**, enter resource name, select **OK**, and select required location from **New resource group location** dropdown list.
-
+    1. To create a new resource group, select **Create new**, enter resource name, select **OK**, and select required location from **New resource group location** dropdown list.
```

Comment:
How is this a mandatory step in the procedure. In the step above we say, select your resource group. In this step we say, create a new resource group. 

Ideally, only 1 of the 2 steps are required. Re-create the list better.

Reply:
Sentence is re-created.